kingpin/actors/rightscale/test/test_base.py

```diff
@@ -10,6 +11,18 @@
 log = logging.getLogger(__name__)
 
 
+class FakeEnsurableRightScaleBaseActor(base.EnsurableRightScaleBaseActor):
```

Comment:
Shouldn't need this. The most base Ensurable actor should have _get and _set state with instant errors:

```python

    @gen.coroutine
    def _get_state(self):
        raise NotImplementedError('_get_state is required for Ensurable')

    @gen.coroutine
    def _set_state(self):
        raise NotImplementedError('_set_state is required for Ensurable')
```